testsupport/wait/host.go

```diff
@@ -1734,7 +1733,7 @@ func (a *HostAwaitility) CreateAPIProxyConfig(t *testing.T, usertoken, proxyURL
 func (a *HostAwaitility) CreateAPIProxyClient(t *testing.T, userToken, proxyURL string) (client.Client, error) {
 	proxyKubeConfig := a.CreateAPIProxyConfig(t, userToken, proxyURL)
 
-	s := scheme.Scheme
+	s := runtime.NewScheme()
```

Comment:
is this actually different than using the variable ? I see it does the same thing, but maybe I've missed something:
```
var Scheme = runtime.NewScheme()
```

Reply:
oh right, this is defined in the scheme.go , never mind , I got confused 🤦‍♂️ .

Reply:
this is creating a new variable, when we use the variable `var scheme`, thats being used at other places, its use of same variable, hence i changed here

Reply:
yeah , right! I've noticed that only after commenting, thanks for looking into it!